README.md

```diff
@@ -1,8 +1,24 @@
-# thoth-unresolved-package-handler
+# thoth-unknown-package-handler
```

Comment:
Let's call this component "The Package Investigator" in the best case, its an agent send out to seek new information on packages, that will yield observations and knowledge, so that it contributes to the protomolecule we are creating...

Reply:
> Let's call this component "The Package Investigator" in the best case, its an agent send out to seek new information on packages, that will yield observations and knowledge, so that it contributes to the protomolecule we are creating...

Simply `investigator` or package-investigator` to be consistent with component naming in Thoth?

Reply:
`Thoth Investigator is an agent sent out to seek new information on packages, that will yield observations and knowledge to Thoth`